server/channels/app/export_test.go

```diff
@@ -156,7 +157,7 @@ func TestCopyEmojiImages(t *testing.T) {
 	os.OpenFile(filePath+"/image", os.O_RDONLY|os.O_CREATE, 0777)
 	defer os.RemoveAll(filePath)
 
-	copyError := th.App.copyEmojiImages(emoji.Id, emojiImagePath, pathToDir)
+	copyError := th.App.copyEmojiImages(request.CTX, emoji.Id, emojiImagePath, pathToDir)
```

Comment:
That is not valid code. You can use `th.Context` instead.

Reply:
> That is not valid code. You can use `th.Context` instead.

Ooops.